db/migrations/0003_alter_skill_race.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.0.2 on 2024-11-26 15:45
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_alter_player_guild'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='skill',
+            name='race',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='db.race'),
```

Comment:
Ensure that the app label 'db' is correct and matches the actual app name where the Race model is defined. If the Race model is in the same app, it should be 'race' instead of 'db.race'.